FinniversKit/Sources/Components/ContractActionView/ContractActionView.swift

```diff
@@ -89,8 +97,12 @@ public class ContractActionView: UIView {
     public func configure(
         with viewModel: ContractActionViewModel,
         trailingImage: UIImage? = nil,
+        trailingImageTopConstant: CGFloat = 0,
```

Comment:
If we rename `trailingImage` to `image`. We can drop the `trailing` prefix in the parameters below, that would however break the API. 

Let me know if I should rename the parameters, I don't feel any particular way about it.

Reply:
It would be nice to drop it, but I think it should be ok to just keep it and optionally create a small refactoring task on it in Jira or something. Because other than that I think it looks good